ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -112,7 +127,7 @@ def _map_sample(self, sample):
             ) from e
 
         if self.json_mode:
-            turns = json.loads(rendered)
+            turns = yaml.safe_load(rendered)
```

Comment:
YAML is a superset of JSON. The issue was that I couldn't figure out how to remove the final comma (`,`) which JSON really strict on. Moving to YAML seemed like and given that we verify the type assumptions, I didn't see why it could hurt.

It does make the naming a bit awkward though: `json_mode` uses YAML. An option is to also change the name to `yaml_mode`.

Reply:
Agree this is a bit awkward, but not sure what the alternative is, I agree the trailing comma issue is annoying. (I guess you could have some sort of second parameter to `json_dump` to control whether to add a comma or not, but even that is ugly.) 

I think it's fine to keep `json_mode` as the name for now though as plain JSON is still supported.